examples/grafana/README.md

```diff
@@ -0,0 +1,31 @@
+# grafana-ocp
+
+This template creates a custom Grafana instance preconfigured to gather Prometheus openshift metrics.
+It is uses OAuth token to login openshift Prometheus.
+
+
+## To deploy grafana
+Note: make sure to have openshift prometheus deployed.
+(https://github.com/openshift/origin/tree/master/examples/prometheus)
+
+1. ```oc create project grafana```
+2. ```oc new-app -f grafana-ocp.yaml```
+3. ```oc expose svc grafana-ocp```
```

Comment:
once the route will be added to the yml file, we will not need to expose the service any more

Reply:
@mrsiano @jeremyeder 

> Lets submit issue 

I understood that in this PR we will add route for http ( port 80, without tls )
and in a later PR update the grafana config file and update route for https ( port 443, with tls )

did I understand correctly ?

Reply:
p.s

1. sttring grafana for https is a setting in the config file [1] ( we have the key/cert from openshift )
 
[1] https://github.com/grafana/grafana/blob/master/conf/defaults.ini#L29